### PR TITLE
Pointer receivers are not required for type CoinIDMap since maps are passed by reference by default

### PR DESCRIPTION
**Pull request by Icelain**

### CODE REVIEW

File: pkg/api/coinID.go [hunk 1]
```diff
@@ -63,21 +63,21 @@ func (c *CoinIDMap) Populate() {
 
 		for _, val := range coinCapData.Data {
 			m.Lock()
-			if _, ok := (*IDMap)[val.Symbol]; ok {
-				(*IDMap)[val.Symbol] = CoinID{
+			if _, ok := IDMap[val.Symbol]; ok {
+				IDMap[val.Symbol] = CoinID{
 					CoinCapID:   val.ID,
 					CoinGeckoID: (*IDMap)[val.Symbol].CoinGeckoID,
 				}
```
Comment (Gituser143) [inline pkg/api/coinID.go]: Type hasn't been changed here.

File: pkg/api/coinID.go [hunk 2]
```diff
@@ -89,12 +89,12 @@ func (c *CoinIDMap) Populate() {
 			symbol := strings.ToUpper(val.Symbol)
 			m.Lock()
 			if _, ok := (*IDMap)[symbol]; ok {
-				(*IDMap)[symbol] = CoinID{
```
Comment (Gituser143) [inline pkg/api/coinID.go]: Type hasn't been changed here too.

**Review by Gituser143 — CHANGES_REQUESTED** [review]
Hi @Icelain! Thank you for your PR! That's a good catch, I don't know how I missed that. Although, you've missed a few changes when altering the pointer type of the map, two places I found them are mentioned in this review.

Additionally, there exists another type with the same issue called `CurrencyIDMap`. You can find this in `pkg/display/utilitywidgets/currency.go`, please modify that too.

File: pkg/api/coinID.go (hunk 2 above)
Comment (Icelain) [reply pkg/api/coinID.go]: I've changed the pointer receivers in CurrencyIDMap as well as the places I had missed them. Could you look at it now?

**Comment by Gituser143** [comment]
The build is still failing, looks like you've missed one place. I think it's the one at line 69 at `pkg/api/coinID.go`.

File: pkg/api/coinID.go (hunk 2 above)
Comment (Icelain) [reply pkg/api/coinID.go]: finally done

**Review by Gituser143 — APPROVED** [review]
@Icelain Looks good! Thank you for your contribution!

**Pull request merged**